aspnet/tutorials/your-first-mac-aspnet.rst

```diff
@@ -1,60 +1,46 @@
 Your First ASP.NET 5 Application on a Mac
 =========================================
 
-By `Steve Smith`_, `Daniel Roth`_
+By `Daniel Roth`_, `Steve Smith`_, `Rick Anderson`_
 
-ASP.NET 5 is cross-platform and can be developed and run on Mac OS X as well as Linux and Windows. See how you can quickly install, scaffold, run, debug, and deploy ASP.NET applications on a Mac.
+ASP.NET 5 is cross-platform; you can develop and run web apps on Mac OS X, Linux and Windows. 
```

Comment:
How about something like: "Let's get started writing our first ASP.NET 5 application on a Mac"

Or: "This article will show you how to write your first ASP.NET 5 application on a Mac"